### ISSUE TITLE
Replace `VecDeque` with `stream::from_iter` in examples

### ISSUE THREAD

**Issue by yoshuawuyts** [async-rs/async-std#430]
Now that https://github.com/rust-lang-nursery/futures-rs/pull/1930 has landed, we're waiting on https://github.com/async-rs/async-std/pull/415 to be merged. Once it's merged we should replace all uses of `VecDeque` with `stream::from_iter` so we're setup for the `futures@0.3.0` release.

This should be a fairly mechanical change, and easy to pick up!

**Comment by k-nasa** [comment]
Will I do this with #415 ?

**Comment by yoshuawuyts** [comment]
@k-nasa I think we should merge that first, and create a follow-up PR to it that ports over the examples. That way we keep moving forward without having a single massive PR that does both things at once. Would you be willing to write the follow-up PR?
